frontend/integration-tests/tests/developer-catalog.scenario.ts

```diff
@@ -42,24 +42,26 @@ describe('Developer Catalog', () => {
 
   it('displays "Jenkins" catalog tile when filter by name: "jenkins"', async() => {
     await catalogPageView.filterByKeyword('jenkins');
+    await crudView.isLoaded();
     expect(catalogPageView.catalogTileFor('Jenkins').isDisplayed()).toBe(true);
   });
 
   it('displays "No Filter Results" page correctly', async() => {
     await catalogPageView.filterByKeyword('NoFilterResultsTest');
-    await catalogPageView.clickFilterCheckbox('kind-ImageStream');
+    await catalogPageView.clickFilterCheckbox('kind-image-stream');
     expect(catalogPageView.catalogTiles.count()).toBe(0);
-    expect(catalogPageView.filterCheckboxFor('kind-ImageStream').isSelected()).toBe(true);
+    expect(catalogPageView.filterCheckboxFor('kind-image-stream').isSelected()).toBe(true);
     expect(catalogPageView.clearFiltersText.isDisplayed()).toBe(true);
 
     await catalogPageView.clearFiltersText.click();
+    await crudView.isLoaded();
     expect(catalogPageView.filterTextbox.getAttribute('value')).toEqual('');
 
-    expect(catalogPageView.filterCheckboxFor('kind-ImageStream').isSelected()).toBe(false);
+    expect(catalogPageView.filterCheckboxFor('kind-image-stream').isSelected()).toBe(false);
     expect(catalogPageView.catalogTiles.count()).toBeGreaterThan(0);
   });
 
-  it('filters catalog tiles by \'Service Class\' Type correctly', async() => {
+  xit('filters catalog tiles by \'Service Class\' Type correctly', async() => {
```

Comment:
it seems that even running tests with `--params.servicecatalog true` still no `serviceclass`

Reply:
We'd need to enable service catalog in CI and switch these tests to use a different broker. Currently console doesn't show any service classes from template service broker by design.

We might just remove this and 'creates a service instance and binding' below. These tests will have to be rewritten either way because they require TSB.

Reply:
Ok, let me remove them

Reply:
they should be removed in the update